### ISSUE TITLE
Cinnanmon Dies 2-3 / Week

### ISSUE THREAD

**Issue by ghost** [linuxmint/cinnamon#11819]
### Distribution

Mint 21.2

### Package version

5.8.4

### Graphics hardware in use

Intel Corporation TigerLake-LP GT2 [Iris Xe Graphics]

### Frequency

Always

### Bug description

Read it all here:

https://termbin.com/pnhsc

### Steps to reproduce

Nothing, it just happens like 2-3 times per week. Thinking if doing a back-up and then change distro.

### Expected behavior

Not to ever happen.

### Additional information

N/A

**Comment by mtwebster** [comment]
Hi, please see https://github.com/linuxmint/cinnamon/issues/11772#issuecomment-1674904774 for troubleshooting steps.

Thanks!

**Comment by itnerdshbg** [comment]
Yes, I have followed those steps but I have nothing. Clean install with XAMPP installed for developing websites, that's it. It has worked for years and now - for the past 3-4 weeks, crashing constantly.
